docs/langref/src/lin_gradients.md

```diff
@@ -1,6 +1,6 @@
 ### Linear Gradients
 
-Gradients allow creating smooth colorful surfaces. They are specified using an angle and a series of
+Linear gradients describe smooth, colorful surfaces. They're specified using an angle and a series of
```

Comment:
Excellent example of an edit that replaces "allowance" with a much better concrete term (describe) 👍